README.md

```diff
@@ -25,6 +25,14 @@ An extension for both Chrome and Firefox that lets you search music, videos, ebo
 4. Extension is added to your browser. Enjoy !!
 5. Launch the extension by clicking the extension's logo or by hitting `Ctrl+Shift+F` from your keyboard.
 
+## Fearch Extension for Google Chrome
+
+	[Download From Chrome Webstore](https://chrome.google.com/webstore/detail/fearch/kalmlbhhgndofalfencmggfncedfajgj?hl=en)
+
+## Fearch Extension for Google Chrome
```

Comment:
Fearch Extension for ~Google Chrome~ Firefox.

Maybe we can change the layout to 

## Add Fearch to your browser
1. [Link for Google Chrome](https://chrome.google.com/webstore/detail/fearch/kalmlbhhgndofalfencmggfncedfajgj?hl=en)
2. [Link for Mozilla Firefox](https://addons.mozilla.org/en-US/firefox/addon/fearchit/?src=userprofile)

Reply:
@PaliwalSparsh  Thanks for the suggestion but i think current layout is Fine

Reply:
I am just curious to know why do you intend to have two subsections for something which falls under the same category ie Adding fearch to your browser. Also, i guess you missed other comment. You have actually put 'Fearch Extension for Google Chrome' in both subheadings.

Reply:
suggested changes have been applied @PaliwalSparsh